roles/pulibrary.solrcloud/tasks/main.yml

```diff
@@ -17,7 +17,7 @@
   tags:
     - mnt
 
-## Mount the share
+## Mount the share - comment this out for solrcloud_staging
```

Comment:
I don't understand this — is there a mechanism to disable this for staging that I'm missing?